dockerfiles/backfill_storage/startup_script.sh

```diff
@@ -28,5 +28,5 @@ if test -z "$VDB_PG_CONNECT"; then
 fi
 
 # Run getStorageValue
-echo "Running backfillStorage from block <NEEDS UPDATE> to <NEEDS UPDATE>"
-./vulcanizedb backfillStorage -s=10739859 -e=10906308 --config config.toml # TODO: update block numbers before next run
+echo "Running backfillStorage from block  10932663 to 10970892"
+./vulcanizedb backfillStorage -s=10932663 -e=10970892 --config config.toml # TODO: update block numbers before next run
```

Comment:
This starting block is the MCD_FLIP_COMP_A_1.1.2 deploy block - 1, and the ending block is the block about when the geth process was restarted with these addresses.